src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -28,14 +23,26 @@ library GPv2Encoding {
     /// @dev An enum describing an order kind, either a buy or a sell order.
     enum OrderKind {Sell, Buy}
 
-    /// @dev The order EIP-712 type hash for the [`Order`] struct (excluding the
-    /// extra `typeHash` field).
+    /// @dev The order EIP-712 type hash for the [`Order`] struct.
+    ///
+    /// This value is pre-computed from the following expression:
+    /// ```
+    /// keccak256(
+    ///     "Order(" +
+    ///         "address sellToken," +
+    ///         "address buyToken," +
+    ///         "uint256 sellAmount," +
+    ///         "uint256 buyAmount," +
+    ///         "uint32 validTo," +
+    ///         "uint32 nonce," +
+    ///         "uint256 tip," +
+    ///         "uint8 kind," +
+    ///         "bool partiallyFillable" +
+    ///     ")"
+    /// );
+    /// ```
     bytes32 internal constant ORDER_TYPE_HASH =
-        // TODO: Replace this with the pre-computed value once the contract is
-        // ready to be deployed.
-        keccak256(
-            "Order(address sellToken,address buyToken,uint256 sellAmount,uint256 buyAmount,uint32 validTo,uint32 nonce,uint256 tip,uint8 kind,bool partiallyFillable)"
-        );
+        hex"70874c19b8f223ec3e4476223f761070db29e881be331cda28425f9079d3a76b";
```

Comment:
This was done to get around:
```src/contracts/libraries/GPv2Encoding.sol:200:31: TypeError: Only direct number constants and references to such constants are supported by inline assembly.
            mstore(dataStart, ORDER_TYPE_HASH)
                              ^-------------^
```

We could also declare a stack variable outside of the assembly, but then we run into too many variables errors.